pkg/workspace/inference/preset-inferences.go

```diff
@@ -191,3 +203,70 @@ func CreatePresetInference(ctx context.Context, workspaceObj *kaitov1alpha1.Work
 	}
 	return depObj, nil
 }
+
```

Comment:
Is there any chance to dedup this method code with the existing code in fine tuning for the similar funcitonality?

Reply:
sure